Cargo.toml

```
@@ -17,9 +17,10 @@ all-features = true
 default = ["http-client"]
 http-client = ["telegram-trait", "dep:ureq", "dep:multipart", "dep:mime_guess", "dep:serde_json"]
 telegram-trait = []
-async-http-client = ["async-telegram-trait", "dep:reqwest", "dep:tokio", "dep:serde_json"]
+async-http-client = ["dep:tokio", "async-telegram-trait", "dep:reqwest", "dep:serde_json"]
```

Can you change this file so it has less diff? Then it's easier to see what actually changed.

For example this can be like it was before:

```suggestion
async-http-client = ["async-telegram-trait", "dep:reqwest", "dep:tokio", "dep:serde_json"]
```